node/api/channels_api_usage.md

```diff
@@ -1563,7 +1565,53 @@ Examples for this would be either opening amount being below the threshold
 defined by `channel_reserve`, or any of `channel_reserve`, `push_amount` or
 `lock_period` being a negative number.
 
+#### Timeout error
+
+In order to prevent zombie states of the state channel that run indefinitely,
+there are a set of timeouts defined by each specific participant [at WebSocket
+connection opening time](#channel-parameters). Violating those is a violation
+of the off-chain protocol and the non-responsive participant is considered for
+being missing. In this case the connection is interrupted and the client is
+expected either to try reaching for the other party and reconnecting or going
+through the solo closing sequence.
```

Comment:
Link to the definition of the sequence would be good.

Reply:
But since this is not handled by the FSM, this shall probably refer to the `ON-CHAIN.md`?

Reply:
Right.